sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -1701,15 +1720,27 @@ class MeanAbsolutePercentageError(BaseForecastingErrorMetricFunc):
         If array-like, values used as weights to average the errors.
         If 'raw_values', returns a full set of errors in case of multioutput input.
         If 'uniform_average', errors of all outputs are averaged with uniform weight.
+    multilevel : {'raw_values', 'uniform_average', 'uniform_average_time'}
+        Defines how to aggregate metric for hierarchical data (with levels).
+        If 'uniform_average' (default), errors are mean-averaged across levels.
+        If 'uniform_average_time', errors are mean-averaged across rows.
+        If 'raw_values', does not average errors across levels, hierarchy is retained.
 
     See Also
     --------
     MedianAbsolutePercentageError
     MeanSquaredPercentageError
     MedianSquaredPercentageError
 
+    Notes
+    -----
+    In some literatures, sMAPE is also known as Adjusted MAPE because
+    ,in some cases, over- and under-forecasts are not really penalised equally. [1]_
+
     References
     ----------
+    .. [1] Goodwin.P and Lawton.R (1999) "On the asymmetry of the symmetric MAPE".
+
     Hyndman, R. J and Koehler, A. B. (2006). "Another look at measures of
```

Comment:
this is not cited in the text, would be worth pointing to this, and add an exact section or page reference.

Reply:
Unfortunately, I dont have access to the full paper. but the paper snippets the author wants to call it modified MAPE. Anyway, for context, I got the paper from the youtube video that was shared by the person who raised this issue.